Wiki/weight-tracker.md

```diff
@@ -1,30 +1,25 @@
-# Weight-tracker
-Weight Tracker is a [Script API](script-api.md) showcase present in the [demo document](document.md).
+# Weight-tracker
 
-[Day notes](day-notes.md) shows (among others) how we have "weight" [promoted attribute](promoted-attributes.md) in the day note [template](template.md). This then aggregates the data and shows a nice chart of weight change in time.
 
-Demo
-----
+The `Weight Tracker` is a [Script API](script-api.md) showcase present in the [demo document](document.md).
 
-![](images/weight-tracker.png)
+By adding `weight` as a [promoted attribute](promoted-attributes.md) to the [Day notes](day-notes.md) through its [template](template.md), you can aggregate the data and plot weight change over time.
 
-How to remove Weight Tracker button from the top bar
-----------------------------------------------------
+![](images/weight-tracker.png)
 
-In the link map of Weight Tracker, there is a note "Button". Open it and delete or comment out its contents. Weight Tracker button will disappear after you close and open the app.
+## Implementation
 
-Implementation
---------------
+The `Weight Tracker` Note in the screenshot above is of type `Render Note`. That type of note doesn't have any useful content itself, rahter it provides a place where a [script](scripts.md) can render its output.
 
-Note "Weight Tracker" in the screenshot above is of type "Render HTML note". Such note doesn't have any useful content itself, the only purpose of it is to provide a place where some [script](scripts.md) can render some output. This script is defined in [relation](attributes.md) `renderNote` - coincidentally it's the Weight Tracker's child `Implementation`.
+Scripts for `Render Notes` are defined in a [relation](attributes.md) called `~renderNote`. In this example, it's the `Weight Tracker`'s child `Implementation`. The Implementation consists of two [code notes](code-notes.md) that contain some HTML and JavaScript respectively, which load all the notes with a `weight` attribute and displays their values in a chart.
 
-This Implementation [code note](code-notes.md) then contains some HTML and JavaScript which loads all the notes with "weight" attribute and displays them in a chart. To actually render chart we're using third party library [chart.js](https://www.chartjs.org/) which is imported as an attachment (it's not built-in into Trilium).
+To actually render the chart, we're using a third party library called [chart.js](https://www.chartjs.org/) which is imported as an attachment, since it's not built into Trilium.
 
-### JS code
+### Code
 
-To get an idea of the script, here's the "JS code" note content:
+Here's the content of the script Note of type `JS Frontend`:
```

Comment:
Here's the content of the script, which is placed in a [code note](code-notes.md) of type `JS Frontend`: